modules/cnv-preparing-nodes-to-enable-dedicated-resources.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-preparing-nodes-to-enable-dedicated-resources_{context}"]
+
+= Preparing to enable dedicated resources
+
+To prepare to enable dedicated resources, the `cpumanager` label must be `true`
+for each node on which you want to schedule virtual machines with
+dedicated resources.
+
+If the `cpumanager` label isn't applied to an available node,
+the workload can not be scheduled.
+
+.Procedure
```

Comment:
I would not detail it here, I would just linked out to https://docs.openshift.com/container-platform/4.3/scalability_and_performance/using-cpu-manager.html - I don't think we should duplicate the documentation which already exists and is maintained somewhere else.

Reply:
looks good now

Reply:
I added a link as you suggested. I have also kept the detailed instructions in a file for myself in case @aburdenthehand feels differently.

Reply:
@jelkosz - in that case, can we cover this simply with a Prerequisite for the UI procedure that says something like:
`* The xref:<xref_relative_filepath_to_cpu-manager.adoc#anchor[CPU Manager] must be running and have the `cpumanager = "true"` label applied to the node before it can be used to schedule virtual machine workloads. `
Where the xref directs to the OCP CPU Manager content?

Reply:
In light of the comment below, recommend removing this module and including the following prerequisite in the UI procedure
`* The xref:<xref_relative_filepath_to_cpu-manager.adoc#anchor[CPU Manager] must be configured on the node. Verify that the node has the `cpumanager = "true"` label before scheduling virtual machine workloads.`
Or words to that effect.

Reply:
Done.